engine/internal/config/config.go

```diff
@@ -191,6 +197,9 @@ type ScalingConfig struct {
 
 	MaxReplicas int `yaml:"maxReplicas"`
 	MinReplicas int `yaml:"minReplicas"`
+
+	MaxScaleUpRate   float64 `yaml:"maxScaleUpRate"`
```

Comment:
Could you add a brief comment on these fields? Some example case that explains the behavior would be useful.